__tests__/producer.integration.test.js

```diff
@@ -122,53 +122,52 @@ describe('Tests while producer is connected', () => {
       kafkaProducer.poll();
     }, 500);
 
-    kafkaProducer.once('delivery-report', (err, report) => {
+    kafkaProducer.once("delivery-report", (err, report) => {
       clearInterval(interval);
 
       expect(err).toBeNull();
       expect(report.topic).toBe(testTopic);
-      expect(typeof report.partition).toBe('number');
-      expect(typeof report.offset).toBe('number');
+      expect(typeof report.partition).toBe("number");
+      expect(typeof report.offset).toBe("number");
 
       done();
     });
 
     kafkaProducer.produce(testTopic, null, message);
   });
 
-  test('Should produce a valid avro message with default null types', (done) => {
+  test("Should produce a valid avro message with default null types", done => {
     const message = {
-      name: 'Bruce Wayne',
+      name: "Bruce Wayne",
       nullValue: 12345
     };
 
     const interval = setInterval(() => {
       kafkaProducer.poll();
     }, 500);
 
-    kafkaProducer.once('delivery-report', (err, report) => {
+    kafkaProducer.once("delivery-report", (err, report) => {
       clearInterval(interval);
 
       expect(err).toBeNull();
       expect(report.topic).toBe(nullTestTopic);
-      expect(typeof report.partition).toBe('number');
-      expect(typeof report.offset).toBe('number');
+      expect(typeof report.partition).toBe("number");
+      expect(typeof report.offset).toBe("number");
 
       done();
     });
 
     kafkaProducer.produce(nullTestTopic, null, message);
   });
 
-  test('Should throw if the message does not match the avro schema', () => {
+  test("Should throw if the message does not match the avro schema", () => {
     const message = {
-      name: 'Bruce Wayne',
-      age: '25',
+      name: 1681,
```

Comment:
Why is this not Bruce Wayne anymore?  Seems like something isnt working right? @FuzzOli87 can you confirm.

Reply:
I changed it so that it had multiple mistakes in it to check that multiple attribute errors were printed. i.e. I made the name into an integer and the age into a string.